public/locales/de/common.json

```diff
@@ -88,6 +88,7 @@
   "vote": "Abstimmung",
   "charts": "Diagramme",
   "liquidity": "Liquidität",
+  "rangeLiquidity": "Bereich Liquidität",
```

Comment:
let's change all of this instances to concentratedLiquidity as it is the term that we use more.

```suggestion
  "concentratedLiquidity": "Bereich Liquidität",
```

Reply:
As we talked offline, we'll move forward with `Liquidity V3` instead Range/Concentrated Liquidity versions. Also moving the `NEW` badge from `DCA` to `Liquidity V3` in our app header